Integrations/integration-ServiceNow.yml

```diff
@@ -1385,5 +1508,11 @@ script:
       description: Filename of uploaded file to override the existing file name in
         entry
     description: Upload new file
+  - name: servicenow-get-groups
+    arguments:
+    - name: name
```

Comment:
i think we should not have the name mandatory, and if absent - return all groups (including to context) like you did before

Reply:
Changed